trials/sample2/services/init-service/application-dev.yml

```diff
@@ -16,11 +16,7 @@ logging:
             function:
               client:
                 ExchangeFunctions: TRACE
-delay-start: 120
-
-http:
-  readTimeOutMs: 30000
-  connectTimeOutMs: 3000
+delay-start: 30
```

Comment:
Anat increased this on purpose. Are you sure it still works with this low number?

Reply:
It was 'working' when I built/deployed but only after restarting - Ill update